Makefile

```diff
@@ -21,4 +21,11 @@ help: ## Display this help screen
 	@grep -E '^[a-zA-Z_-]+:.*?## .*$$' $(MAKEFILE_LIST) | sort | awk 'BEGIN {FS = ":.*?## "}; {printf "\033[36m%-30s\033[0m %s\n", $$1, $$2}'
 
 docker:
-	@docker build -f build/Dockerfile -t hitachivantaraedge/modbus-service .
\ No newline at end of file
+	@docker build -f build/Dockerfile -t hitachivantaraedge/modbus-service .
```

Comment:
we should not check in vendor folder here.   Build is thru circleCI which can download vendor files during build time